diffsims/sims/diffraction_simulation.py

```diff
@@ -82,6 +151,9 @@ def calibration(self):
 
     @calibration.setter
     def calibration(self, calibration):
+        if calibration is None:
+            self._calibration = calibration
+            return
```

Comment:
Can we avoid the `return`? Perhaps just change `calibration` under any `True` condition, and then just set it at the end?

```python
if calibration is None:
    pass
elif np.allclose(calibration, 0):
   ...
elif isinstance():
   ...
elif len():
   ...
else:
   ...
self._calibration = calibration
```

Reply:
Good idea